test/lib/sdkdriver/src/browser.ts

```diff
@@ -0,0 +1,39 @@
+import DMP from './dmp';
+import WSClientTransport from './transport/websocket/client';
+
+function getUrlParams(): Map<string, string> {
+  const serializedParams: string = window.location.search.substring(1);
+  return serializedParams.split('&').reduce((params, nvpair) => {
+    const [ name, value ] = nvpair.split('=').map(decodeURIComponent);
+    return params.set(name, value);
+  }, new Map());
+}
+
+function loadSdk(url: string): Promise<void> {
```

Comment:
nitpick: although this is fine for now, we should maybe be more generic. we could load an arbitrary module instead of an sdk.